### PR TITLE
test

### PR DESCRIPTION
**Pull request by zhangkanqi**

### CODE REVIEW

**Comment by zhangkanqi** [comment]
增加请求幂等

**Comment by zhangkanqi** [comment]
增加请求幂等

File: scripts/myConfig.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+git clone https://github.com/grpc/grpc-go.git $GOPATH/src/google.golang.org/grpc
+git clone https://github.com/golang/net.git $GOPATH/src/golang.org/x/net
+git clone https://github.com/golang/text.git $GOPATH/src/golang.org/x/text
+git clone https://github.com/google/go-genproto.git $GOPATH/src/google.golang.org/genproto
+git clone https://github.com/golang/sys.git $GOPATH/src/golang.org/x/sys
+git clone https://github.com/protocolbuffers/protobuf-go.git $GOPATH/src/google.golang.org/protobuf
+cd $GOPATH/src/
+go install google.golang.org/grpc
+```
+安装grpc的时候报错缺少哪些依赖包，git clone相应的依赖包。
```
Comment (yunxiao3) [inline scripts/myConfig.md:73]: accept the configuration document changes

**Review by yunxiao3 — APPROVED** [review]
accept the configuration document changes, add more details

**Review by yunxiao3 — COMMENTED** [review]
.idea was removed

**Review by yunxiao3 — COMMENTED** [review]
None

**Pull request closed without merging**